package.json

```diff
@@ -70,80 +70,84 @@
 		"source-map-loader": "4.0.1",
 		"terser-webpack-plugin": "5.3.9",
 		"uglify-js": "^3.17.4",
-		"uuid": "9.0.1",
+		"uuid": "8.3.2",
```

Comment:
let's not downgrade `uuid` [again](https://core.trac.wordpress.org/ticket/58623#comment:37) 😅  - it keeps happening because the Gutenberg version is still 8.3.2. Is there anything blocking us from updating it on the Gutenberg side? Cc @gziolo

Reply:
I was surprised to learn I updated it 3 years ago to `8.3.0` in Gutenberg... 😅 

I don't see any blockers for upgrading to the latest version after reading the [BREAKING CHANGES](https://github.com/uuidjs/uuid/blob/main/CHANGELOG.md#-breaking-changes) section for `v9.0.0`. Let's do it if that helps.

Reply:
I've opened a Gutenberg PR to update `uuid` to 9.0.0: https://github.com/WordPress/gutenberg/pull/54725